src/main/java/org/geysermc/discordbot/listeners/ErrorAnalyzer.java

```diff
@@ -108,7 +108,7 @@ public void onMessageReceived(MessageReceivedEvent event) {
                 groups[i] = groups[i] == null ? "" : groups[i]; // Replace nulls with empty strings
             }
 
-            url = String.format(logUrlPatterns.get(regex), groups);
+            url = String.format(logUrlPatterns.get(regex), groups); // Casting this to Object breaks something, don't do it
```

Comment:
We should disable the inspector warning along with this comment

Reply:
Casting to Object[] should be sufficient right?